crates/roc_host/src/roc.rs

```diff
@@ -945,3 +886,89 @@ pub extern "C" fn roc_fx_tempDir() -> RocList<u8> {
 
     RocList::from(path_os_string_bytes.as_slice())
 }
+
+pub struct RocServer {
+    pub model: roc_std::SendSafeRocList<u8>,
+    pub captures: roc_std::SendSafeRocList<u8>,
+}
+
+unsafe impl Sync for RocServer {}
+
+pub fn call_roc_init() -> RocServer {
+    extern "C" {
+        fn roc__forHost_1_exposed_generic(_: *mut u8);
+        fn roc__forHost_1_exposed_size() -> usize;
+        fn roc__forHost_0_caller(flags: *const (), captures: *mut u8, model: *mut u8);
+        fn roc__forHost_0_size() -> usize;
+    }
+
+    unsafe {
+        let captures_size = roc__forHost_1_exposed_size();
+        let mut captures = RocList::from_raw_parts(
+            roc_alloc(captures_size, 0) as *mut u8,
```

Comment:
I get that alignment doesn't truly matter cause we don't use it currently, but an alignment of `0` is definitely wrong here.